opwen_email_server/utils/email_parser.py

```diff
@@ -110,10 +190,12 @@ def _fetch_image_to_base64(image_url: str) -> Optional[str]:
         return None
 
     image_content = b64encode(response.content).decode('ascii')
-    return 'data:{};base64,{}'.format(image_type, image_content)
+    image_content = _change_image_size(image_content)
+    base64 = 'data:{};base64,{}'.format(image_type, image_content)
```

Comment:
`base64` is a bit misleading as a name here since technically `image_content` is also base64 encoded. Either call this `data_uri` or just inline the variable into the return statement.